File: app/src/main/java/com/kidssaveocean/fatechanger/highScores/Country.java
package com.kidssaveocean.fatechanger.highScores;

import android.support.annotation.NonNull;

public class Country implements Comparable<Country> {
    public String name;
    public int numOfLetters;
    public int rank;

    public Country (String name, int numOfLetters) {
        this.name = name;
        this.numOfLetters = numOfLetters;
    }

    @Override
    public int compareTo(@NonNull Country otherCountry) {
        return otherCountry.numOfLetters - numOfLetters;
    }
}

File: app/src/main/java/com/kidssaveocean/fatechanger/highScores/Country.kt
package com.kidssaveocean.fatechanger.highScores

class Country(var name: String, var numOfLetters: Int) : Comparable<Country> {
    var rank: Int = 0

    override fun compareTo(otherCountry: Country) = otherCountry.numOfLetters - numOfLetters

}
